### ISSUE TITLE
О составе технологической платформы

### ISSUE THREAD

**Issue by MisteryX** [ppdemocracy/manifesto#1]
Приветствую, рад что кто-то с ресурсами занялся этой проблемой... Уже несколько лет вынашиваю идею аналогичной платформы, хотел поделиться идеями.
Решать проблему доверия людей к системе с помощью блокчейн - это, конечно, не плохо (при условии простоты и доступности инструментов проверки цепочки реестра), но для реализации прямой демократии понадобится ведь не только платформа для голосования... Важен также и процесс подготовки законопроектов, общественного планирования, бюджетирования, оперативного принятия решений, контроля исполнения, реальные количественные оценки авторитета так называемых экспертов, разбора инцидентов, и, наконец, самое главное - это оперативная активная обратная связь для всех подсистем (которая сейчас отсутствует напрочь). Ни один человек не совершенен, и общепринятая практика долговременных лидеров/экспертов зарекомендовала себя на протяжении истоии человечества крайне деструктивно. Крайне желательно, чтобы лидеры определялись и сменялись динамически по ходу конкретных проектов в зависимости от их реальной вовлеченности и полезности, а все подсистемы были зарезервированы и имели открытый исходный код и контроллируемый состав инфраструктуры, реестров и баз данных. В противном случае, добиться декларируемых целей, а именно принятия самоуправляющих решений обществом, будет крайне затруднительно...

**Comment by olegart** [comment]
Вообще говоря, партии лидерского типа — это специфичная особенность России и иных «развивающихся рынков», и она действительно деструктивна, т.к. порождает партии с авторитарным управлением, которые после прихода к власти этот авторитаризм внедряют и на уровне государства. В России такие партии — все, от властных до оппозиционных.

Мы такой тип как раз строить не планируем.

**Comment by MisteryX** [comment]
Хорошо, буду рад поучаствовать в проекте, когда появится кодовая база...

**Comment by Pavaila** [comment]
Не формирователем повестки дня, а контролером безопасности системы все равно будет какой-то один человек, пусть даже цифровая система будет "саморегулируемой". Найдется человек с аналоговым рубильником, типа караул устал и света нет. В начале может все и будет гладко, но если не будет персонифицированной "совести нации" начнется внутренняя борьба за кнопку, а потом и внешняя. Должен быть настоящий "гарант". Это не означает его априорной пожизненной несменяемости, но это должна быть особенная и независимая фигура, свою роль он должен осознавать миссией как хранитель народного храма. Я думаю такие люди существуют. И тогда он выполнит что должен, и тогда идея в принципе реализуема. Иначе война за кнопку.

**Comment by ghost** [comment]
Чтобы не было войны за кнопку исходный текст системы должен быть открыт и данные синхронизироваться. И если "совесть нации" погибнет, то  быстро появится новый боец, который поднимет упавший флаг системы.

**Comment by MisteryX** [comment]
Если система действует в интересах большинства в обществе, то она априори устойчива к внутренним конфликтам. Единственное зачем нужны "контролеры безопасности" (специалисты) так это для защиты от внешних угоз и организации обороны общества в условиях войны. И в военное время, они, конечно, должны брать на себя бремя принятия оперативных решений за большинство, ради выживания общества, но не более того.

**Comment by ghost** [comment]
гос органы имеют конфликт с  народом, поэтому вопрос как защитить идею очень правильный. А идею и технологию можно защитить только массовой распространённостью.

**Comment by Pavaila** [comment]
> Если система действует в интересах большинства в обществе, то она априори устойчива к внутренним конфликтам

Нет, это грубая натяжка, нет в этом утверждении ни места слову "априори", ни причинно-следственной связи. Ну действует система в интересах большинства, но не в интересах же всего общества. Появляется активное меньшинство, оно раздражено и недовольно вашей системой, оно делает революцию и ставит свои идеалы справедливости выше, потому что голосовалка хоть и работала на большинство, но это большинство не консолидировано - сегодня вы в него входите, а завтра решение не в вашу пользу и вы в меньшинстве. Так раз 5-10 вы проиграете и первым побежите ломать эту хрень.

**Comment by innovait-rus** [comment]
Похоже я играю в тропико5. Только в текстовом виде. И это заметьте.  5 человек максимум.

**Comment by Pavaila** [comment]
Что тут можно сказать - заметно что ни кто из присутствующих серьезно не думал над темой, не защищал диссертаций. Даже разместивший манифест автор еще плохо понимает о чем говорит.  Остается не опускать руки, настроиться на долгий путь и направить максимум усилий на теоретическую базу, на фундамент для будущего здания.

**Comment by innovait-rus** [comment]
> Что тут можно сказать - заметно что ни кто из присутствующих серьезно не думал над темой

А вы защищали? Можете выложить в открытый доступ и указать ссылку? Почитаю с интересом.

**Comment by ryehrgrwqf** [comment]
Кто сильнее , тот и прав. Гопник - бъёт скрипача, остальные 300 чел стоят и смотрят , ковыряясь в носу. Так было и так будет. Смена технологического уклада неизбежно влечёт за собой смену уклада общественного.
 Читая ваши комментарии господа и в целом наблюдая как поднялся уровень дискуссии в восточной европе я все больше и больше  убеждаюсь лишь в одном. Сообщество  разработчиков эволюционным путём подберёт такой  алгоритм распределения ответственности и принятия решений , который   отмудохает старую систему как гопник очкарика. Бес шансов.  Такова жизнь.  Эмергенция неизбежна. Будущее беспощадно))))

**Comment by ucmok** [comment]
> Крайне желательно, чтобы лидеры определялись и сменялись динамически по ходу конкретных проектов в зависимости от их реальной вовлеченности и полезности, а все подсистемы были зарезервированы и имели открытый исходный код и контроллируемый состав инфраструктуры, реестров и баз данных. В противном случае, добиться декларируемых целей, а именно принятия самоуправляющих решений обществом, будет крайне затруднительно...

В [ТЗ - «Технологии распределенного реестра и смарт-контрактов в качестве инструментов реализации проекта триадного самоуправляемого социума»](https://github.com/ppdemocracy/manifesto/files/4113653/github.docx) в разделе «Иерархическая сетевая система формирования трех блоков триады» описана система, в которой изначально заложена технология сменяемости лидера иерархии (члена иерархии, имеющего доступ к ее Казне) и принципы, в соответствии с которыми этим лидером может стать любой ее член - в частности, благодаря способности членов социума давать оценку общественно-значимой деятельности другим членам социума. Еще более конкретно - предлагается создать такую криптовалюту, которую можно было бы посылать тому или иному члену социума, изменив ее знак на противоположный. Т.е. по сути у членов социума появляется возможность штрафовать любого начальничка - посылая ему отрицательную криптовалюту, уменьшающую сумму в его криптокошельке.

**Comment by ucmok** [comment]
>  контролером безопасности системы все равно будет какой-то один человек, пусть даже цифровая система будет "саморегулируемой".  

Предлагаю ознакомиться с триадным принципом самоуправления системы - это когда три силы попарно циклично контролируют друг друга. 
Цитирую из [ТЗ](https://github.com/ppdemocracy/manifesto/files/4113670/github.docx):
Устойчивость системы, состоящей из трех элементов, контролирующих друг друга описанным образом, можно доказать разными способами, – и «на пальцах», и решив систему дифференциальных уравнений (СДУ) в частных производных (трехмерный аналог СДУ Лотки-Вольтерры). 
Дадим самое простое доказательство. 
Положим, вследствие внутренней флуктуации (или внешнего воздействия) Первый элемент усилился. Он теперь сильнее контролирует, сдерживает, «зажимает» Второй элемент, который в результате слабеет и потому ослабляет свой контроль над Третьим элементом, который по этой причине усиливается и начинает крепче «прижимать» Первый элемент, который тем самым ослабляется и возвращается в свое прежнее состояние – в состояние динамического равновесия в триаде.

**Comment by ucmok** [comment]
> > Что тут можно сказать - заметно что ни кто из присутствующих серьезно не думал над темой
> 
> А вы защищали? Можете выложить в открытый доступ и указать ссылку? Почитаю с интересом.

Защищал я, положим, возможность использование приповерхностных акустических волн для повышения частотных характеристик акусто-электронных устройств, но тему о которой идет речь, разминаю уже не первый год, и ссылку могу предоставить. Если коротко - 
[ТЗ](https://github.com/ppdemocracy/manifesto/files/4113678/github.docx) построения самоуправляемого социума, там на первой странице - ссылка на гораздо более подробную книгу.

**Comment by MisteryX** [comment]
Платформа в целом и каждый из ее модулей должны быть предельно просты, открыты и понятны каждому члену общества, также как и базовые принципы системы. Сложное не работает в больших масштабах - работает только простое. 
В основе должен быть простой и понятный базовый принцип, с которым будет согласно абсолютное большинство. Например - наивысший приоритет интересов большинства членов общества. Если закрепить это в фундаменте системы, то можно говорить об устоичивости и низком уровне противоречивости. Для недовольного, принимаемыми решениями, меньшинства можно создать возможность создать своё собственное обособленное общество на той же открытой платформе, но на нейтральных территориях, где их интересы будут среди большинства членов общества.
